### PR TITLE
New sbs api docs

### PR DESCRIPTION
**Pull request by ryanbrockman**
Adding a new section with docs for the Simple Backup API

### CODE REVIEW

File: Simple Backup/delete-server-policy.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "sticky": "false"
+}}}
+
+Delete a Server Policy under an Account Policy. Calls to this operation must include a token acquired from the authentication endpoint. See the [Login API](../Authentication/login.md) for information on acquiring this token.
+
+### When to Use It
+
+Use this API operation when you want to delete an existing Server Policy.
+
+## URL
```
Comment (srveit) [inline Simple Backup/delete-server-policy.md]: Change to `###`

File: Simple Backup/delete-server-policy.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Structure
+
+    DELETE https://api.backup.ctl.io/clc-backup-api/api/accountPolicies/{accountPolicyId}/serverPolicies/{serverPolicyId}
+
+### Example
+
+    DELETE https://api.backup.ctl.io/clc-backup-api/api/accountPolicies/5fde14a2-fa9d-4376-9f01-59429d02a959/serverPolicies/085384ce-200e-4205-8a13-ae1d0bdc71cd
+
+## Request
```
Comment (srveit) [inline Simple Backup/delete-server-policy.md]: Change to `###`

File: Simple Backup/create-policy.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  "sticky": "true"
+}}}
+
+Creates a new backup policy associated with the account. Calls to this operation must include a token acquired from the authentication endpoint. See the [Login API](../Authentication/login.md) for information on acquiring this token.
+
+### When to Use It
+
+Use this API operation when you want to create a brand new backup policy.
+
+## URL
```
Comment (srveit) [inline Simple Backup/create-policy.md]: Change to `###`

File: Simple Backup/create-policy.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+### Structure
+
+    POST https://api.backup.ctl.io/clc-backup-api/api/accountPolicies
+
+### Example
+
+    POST https://api.backup.ctl.io/clc-backup-api/api/accountPolicies
+
+## Request
```
Comment (srveit) [inline Simple Backup/create-policy.md]: Change to `###`

**Comment by srveit** [comment]
@ryanbrockman change all `##` to `###`

**Comment by ryanbrockman** [comment]
Stephen - In looking at the other API docs, it seems the standard for the URL, Request, and Response headers is to be ##.  Just wanted to clarify if ### is really what you would like me to do, cause it seems as though it will make our docs look different from the others.

**Pull request merged**